cirq/study/trial_result.py

```diff
@@ -120,15 +124,30 @@ def __init__(self, *,  # Forces keyword args.
             repetitions: The number of times the circuit was sampled.
         """
         self.params = params
-        self.measurements = measurements
-        self.repetitions = repetitions
+
+        # Convert to a DataFrame with columns as measurement keys, rows as
```

Comment:
Done. Had to fix a bunch of uses and might break any external usage of TrailResult constructor.

Reply:
I know this is already merged, but this would have been an easy place to put in a deprecation warning for one point release and then remove it in the following point release